src/cryptography/hazmat/backends/openssl/ec.py

```diff
@@ -200,6 +200,23 @@ def private_numbers(self):
             public_numbers=self.public_key().public_numbers()
         )
 
+    def private_bytes(self, encoding, format, encryption_algorithm):
+        evp_pkey = self._backend._lib.EVP_PKEY_new()
+        assert evp_pkey != self._backend._ffi.NULL
+        evp_pkey = self._backend._ffi.gc(
+            evp_pkey, self._backend._lib.EVP_PKEY_free
+        )
+        res = self._backend._lib.EVP_PKEY_set1_EC_KEY(evp_pkey, self._ec_key)
```

Comment:
Using the OpenSSL API, maybe not, but I think we generally do need access to the low level key objects for the other operations we implement.